### ISSUE TITLE
error

### ISSUE THREAD

**Issue by NorwyysYT** [GaijinEntertainment/fmod_studio_warthunder_for_modders#79]
When i try to open the project in FMOD it just says that it cannot open becouse required folders are missing or something :(

**Comment by RadonSound** [comment]
Hello,
What does the error box says? or what folders are missing?

**Comment by NorwyysYT** [comment]
- Master Sandbox Folder
- Master Event Folder
- Profiler Session Folder
- Master Asset Folder
- Mixer Master
- Mixer
- Master Tag Folder
- Master Parameter Preset Folder
- Master Effect Preset Folder
- Workspace
- Snapshot List
- Master Bank Folder
those are missing, or so it says.

**Comment by RadonSound** [comment]
Does this happen on a fresh project?

**Comment by NorwyysYT** [comment]
No, this does not happen on a new/fresh project at least as i know.

**Comment by NorwyysYT** [comment]
@Uchuujin01 how do i view the project folder?

**Comment by Uchuujin01** [comment]
@NorwyysYT Hello! I'm sorry, I missed your comment somehow. Hope you solved this issue?
